[15/06/2019 14: 36:08][DEBUG] docker.utils.config[find_config_file] [config.py],21 Trying paths: ['/home/fabio/.docker/config.json', '/home/fabio/.dockercfg']
[15/06/2019 14: 36:08][DEBUG] docker.utils.config[find_config_file] [config.py],28 No config file found
[15/06/2019 14: 36:08][DEBUG] docker.utils.config[find_config_file] [config.py],21 Trying paths: ['/home/fabio/.docker/config.json', '/home/fabio/.dockercfg']
[15/06/2019 14: 36:08][DEBUG] docker.utils.config[find_config_file] [config.py],28 No config file found
[15/06/2019 14: 36:08][WARNING] werkzeug[_log] [_internal.py],122  * Debugger is active!
[15/06/2019 14: 36:08][INFO] werkzeug[_log] [_internal.py],122  * Debugger PIN: 392-150-906
[15/06/2019 14: 36:17][DEBUG] docker.utils.config[find_config_file] [config.py],21 Trying paths: ['/home/fabio/.docker/config.json', '/home/fabio/.dockercfg']
[15/06/2019 14: 36:17][DEBUG] docker.utils.config[find_config_file] [config.py],28 No config file found
[15/06/2019 14: 36:17][DEBUG] docker.utils.config[find_config_file] [config.py],21 Trying paths: ['/home/fabio/.docker/config.json', '/home/fabio/.dockercfg']
[15/06/2019 14: 36:17][DEBUG] docker.utils.config[find_config_file] [config.py],28 No config file found
[15/06/2019 14: 36:18][INFO] werkzeug[_log] [_internal.py],122  * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
[15/06/2019 14: 36:18][INFO] werkzeug[_log] [_internal.py],122  * Restarting with stat
[15/06/2019 14: 36:18][DEBUG] docker.utils.config[find_config_file] [config.py],21 Trying paths: ['/home/fabio/.docker/config.json', '/home/fabio/.dockercfg']
[15/06/2019 14: 36:18][DEBUG] docker.utils.config[find_config_file] [config.py],28 No config file found
[15/06/2019 14: 36:18][DEBUG] docker.utils.config[find_config_file] [config.py],21 Trying paths: ['/home/fabio/.docker/config.json', '/home/fabio/.dockercfg']
[15/06/2019 14: 36:18][DEBUG] docker.utils.config[find_config_file] [config.py],28 No config file found
[15/06/2019 14: 36:18][WARNING] werkzeug[_log] [_internal.py],122  * Debugger is active!
[15/06/2019 14: 36:18][INFO] werkzeug[_log] [_internal.py],122  * Debugger PIN: 392-150-906
